components/multi-platform-controller/production-downstream/kustomization.yaml

```diff
@@ -1,8 +1,17 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+
+namespace: multi-platform-controller
+
 resources:
```

Comment:
I propose to make the deploy resources tag distinctive too, that will allow to test changes on staging/development

Reply:
yeah... I meant to do that... but it seems it going to be ugly... hold on...

Reply:
Please check now